src/staging/cora_mapper_validation_temp_delete.py

```diff
@@ -0,0 +1,51 @@
+import logging
+import pandas as pd
+
+from src.utils.wrappers import exception_wrap
+
+validation_logger = logging.getLogger(__name__)
+
+@exception_wrap
+def validate_cora_df(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Validates cora mapper df:
+    1. Checks if the DataFrame has exactly two columns.
+    2. Checks if the column headers are "statusencoded" and "form_status".
+    3. Checks the validity of the columns contents.
+    Args:
+        df (pd.DataFrame): The input DataFrame containing "statusencoded"
+        and "form_status" columns.
+    Returns:
+        df (pd.DataFrame): with cols changed to string type
+    """
+    try:
+        # Check if the dataframe has exactly two columns
+        if df.shape[1] != 2:
+            raise ValueError("DataFrame must have exactly two columns")
+
+        # Check if the column headers are "statusencoded" and "form_status"
+        if list(df.columns) != ["statusencoded", "form_status"]:
+            raise ValueError("Column headers must be 'statusencoded' and 'form_status'")
+
+        # update data types of mapper to str
+        df["statusencoded"] = df["statusencoded"].astype(str)
```

Comment:
The data type casting is done using the other validation function in staging_main. These 2 lines are repetitions and cause errors with nans